dev-util/drone/files/droned.init

```diff
@@ -0,0 +1,75 @@
+#! /bin/sh
```

Comment:
Documentation is here: https://wiki.gentoo.org/wiki/Handbook:X86/Working/Initscripts#Writing_initscripts